exporter/elasticsearchexporter/factory.go

```diff
@@ -91,9 +91,13 @@ func createDefaultConfig() component.Config {
 				MinSizeItems: 5000,
 			},
 			MaxSizeConfig: exporterbatcher.MaxSizeConfig{
-				MaxSizeItems: 10000,
+				MaxSizeItems: 0,
 			},
 		},
+		Flush: FlushSettings{
+			Bytes:    5e+6,
```

Comment:
nit: maybe consider moving these defaults to some constant vars at some point. (not for this PR)